frontend/src/styles/Root.css

```diff
@@ -40,3 +35,61 @@
 .read-the-docs {
   color: #888;
 }
+
+body{
+  background: linear-gradient(45deg, gray, white);
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.welcomeWrapper{
+  display: flex;
+  width: 100vw;
+  justify-content: center;
+  flex-wrap: wrap
+}
+
+.title{
+  width: 100%;
+  font-size: 40px;
+  justify-content: center;
+  text-align: center;
+	color: white;
+	margin-bottom: 0;
+	padding-bottom: 0;
+}
+
+.text{
+	font-size: 105px;
+}
+
+.description{
+  width: 100%;
+  text-align: center;
+  font-size: 50px;
+	font-weight: bold;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  margin-bottom: 0.4em;
+	color: white;
+}
+
+.signUpButton{
+  background: linear-gradient(360deg, #8e9eab, #eef2f3);
+  padding: 1em;
+	padding-left: 2em;
+	padding-right: 2em;
+  border-radius: 8px;
+  font-size: 25px;
+	border: 0px;
+}
+
+#error-page{
+	display: flex;
+	justify-content: center;
+	width: 100vw;
+	flex-wrap: wrap;
+}
+.er{
```

Comment:
It's generally better to be more verbose. I would rename this to `errorText` as it's used for `<h1>` and `<p>` as far as I can tell